### ISSUE TITLE
Building the BioNJ trees

### ISSUE THREAD

**Issue by KYQiu21** [stephaneguindon/phyml#181]
Dear all,

Is it possible to infer only the BioNJ trees given a sequence alignment? If so, what is the parameter setting for this? Thanks.

Best,
K.

**Comment by stephaneguindon** [comment]
`./phyml -i your_sequences -o none` should do the trick

**Comment by stephaneguindon** [comment]
Did you use the '-d aa' option?

**Comment by KYQiu21** [comment]
Yes! ./phyml -i example_dataset -o none -d aa gives the error report I posted above

**Comment by stephaneguindon** [comment]
Could you please indicate which version of PhyML you're running, which OS and how you compiled?
